### ISSUE TITLE
[Bug]: Application menu dropdown menus are tiled

### ISSUE THREAD

**Issue by panbroggi** [gikari/bismuth#95]
### Summary

I am using the application menu. When I select a category, the available choices are treated as a new window and tiled.

### Steps to Reproduce

Press the menu icon or press the META key. Position the cursor on one of the available categories.

### Expected behavior

The choices should appear next to the selected category.

### Screenshots

![Bug](https://user-images.githubusercontent.com/76168057/136024727-9f3bcb53-feb0-42c2-9d1c-684012951c0c.png)

### Bismuth version

1.01

### KDE Plasma version

5.22.5

### The platform KWin is running on

Wayland

### Additional context

I am using KaOS Linux with KDE Framework 5.86.0 and My Kernel is 5.13.11.

**Comment by gikari** [comment]
I am not sure, but generally on Wayland some popups were broken (displayed as regular windows) until recent updates in Plasma 5.23. So this may be an upstream KWin bug - it may be fixed in 5.23, which is coming soon.

**Comment by panbroggi** [comment]
Thanks for the fast reply. I will temporarily switch to the Kickoff menu - which has no problem. Once the update is released I will be back.

**Comment by panbroggi** [comment]
Hi! I updated to Plasma 5.23.0 and there is the same problem. Is there a way to find the name of the popups so that they're excluded by tiling?

**Comment by gikari** [comment]
Yes, when you're adding a KWin rule there is a button, that lets you select the window you like. 


![Screenshot_20211016_173855](https://user-images.githubusercontent.com/14205339/137591610-a6e7bd34-c320-490e-991f-11f5017d9902.png)

After you've made the selection, it then prompts to select the properties you're interested in. 

![Screenshot_20211016_174208](https://user-images.githubusercontent.com/14205339/137591705-8ab5fe39-8593-482a-884a-9214e6f97882.png)


Please say what the rules are, so they could be added to the default configuration.
